conan/cli/commands/upload.py

```diff
@@ -24,17 +26,31 @@ def format_upload(item):
                     v.pop("timestamp", None)
                     v.pop("files", None)
                     v.pop("upload-urls", None)
+                    uploaded_value = v.pop("uploaded", None)
                     upload_value = v.pop("upload", None)
-                    if upload_value is not None:
-                        msg = "Uploaded" if upload_value else "Skipped, already in server"
+                    if uploaded_value is not None and upload_value is not None:
                         force_upload = v.pop("force_upload", None)
-                        if force_upload:
-                            msg += " - forced"
+                        if uploaded_value:
+                            msg = "Uploaded"
+                            if force_upload:
+                                msg += " - forced"
+                        elif not upload_value:
+                            msg = "Skipped, already in server"
+                        else:
+                            msg = "Not uploaded"
```

Comment:
Seems unrelated to the current PR? If it is a fix, it should be in a separate PR probably